.vitepress/config.ts

```diff
@@ -15,108 +15,108 @@ export default defineConfig({
       '/': [
         {
           text: 'Overview',
-          link: '/Overview.md',
+          link: '/overview.md',
           items: [
             {
               text: 'How FIB works',
-              link: '/Overview/How_FIB_works.md',
+              link: '/overview/how_fib_works.md',
```

Comment:
kebab-case considers `how-fib-works.md`, not `how_fib_works.md`, relevant for most entries

Reply:
The actual file is still `Overview/How_FIB_works.md`, so this link should be dead. Also, it is not necessary to specify `.md` here (and it is recommended to not specify it in any links in markdown files themselves).